library/src/main/java/com/novoda/downloadmanager/DownloadBatch.java

```diff
@@ -370,6 +373,28 @@ void delete() {
                          + ", should be deleting");
     }
 
+    private void deleteDownloadDirs() {
+        BatchStorageRoot batchStorageRoot = BatchStorageRoot.with(() -> downloadBatchStatus.storageRoot(), downloadBatchStatus.getDownloadBatchId());
```

Comment:
You can use a method reference for the `downloadBatchStatus.storageRoot`.

Reply:
ok